azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go

```diff
@@ -909,6 +930,11 @@ func resourceArmKubernetesClusterRead(d *schema.ResourceData, meta interface{})
 			return fmt.Errorf("setting `default_node_pool`: %+v", err)
 		}
 
+		kubeletIdentity := flattenKubernetesClusterIdentityProfile(props.IdentityProfile)
+		if err := d.Set("kubelet_identity", kubeletIdentity); err != nil {
+			return fmt.Errorf("setting `kubelet_identity`: %+v", err)
+		}
```

Comment:
@tombuildsstuff Should I rename it to ` flattenKubernetesClusterKubeletIdentity` or leave it as is?

Reply:
this is fine as-is 👍